service/kernel/service_topo.cpp

```diff
@@ -2023,6 +2032,11 @@ unsigned getL1CacheSize()
     return 32 * 1024;
 }
 
+unsigned getL2CacheSize()
```

Comment:
This path of code is related to Mac, right?
If so - I propose to set this to 256kB, because it's more relevant for desktop cpus (main use case for Mac).